rules-tests/Php80/Rector/Class_/AnnotationToAttributeRector/Fixture/inheritancetype_with_discriminator_column_using_constants_map.php.inc

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping\Entity;
+
+/**
+ * @Entity
+ */
+class Employee extends Person
+{
+    public const TYPE = 'employee';
+}
+
+?>
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Doctrine\Tests\Set\DoctrineORM29Set\Fixture;
+
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\InheritanceType;
+use Doctrine\ORM\Mapping\DiscriminatorColumn;
+use Doctrine\ORM\Mapping\DiscriminatorMap;
+
+/**
+ * @Entity
+ * @InheritanceType("SINGLE_TABLE")
+ * @DiscriminatorColumn(name="discr", type="string")
+ * @DiscriminatorMap({Person::TYPE = Person::class, Employee::TYPE = Employee::class})
```

Comment:
Could you make test as simple as possible? Ideally this line only, so we can focus only on one problematic line at a time